templates/core/terraform/airlock/locals.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  # STorage AirLock EXternal
+  airlock_external_import_storage_name = lower(replace("stalexim${var.tre_id}", "-", ""))
+  # STorage AirLock InProgress IMport
+  airlock_in_progress_import_storage_name = lower(replace("stalipim${var.tre_id}", "-", ""))
+  # STorage AirLock REJected IMport
+  airlock_rejected_import_storage_name = lower(replace("stalrejim${var.tre_id}", "-", ""))
+  # STorage AirLock ACCepted EXPort
+  airlock_accepted_export_storage_name = lower(replace("stalaccexp${var.tre_id}", "-", ""))
+
+  egst_inprogress_import_sys_topic_name = "egst-in-prog-import-${var.tre_id}"
+  egst_rejected_import_sys_topic_name   = "egst-rejected-import-${var.tre_id}"
+  egst_accepted_export_sys_topic_name   = "egst-accepted-export-${var.tre_id}"
+  egt_scan_result_topic_name            = "egt-scan-res-${var.tre_id}"
+  egt_update_status_topic_name          = "egt-update-status-${var.tre_id}"
+  egt_status_changed_topic_name         = "egt-status-changed-${var.tre_id}"
```

Comment:
1. Too much shorthand on the TF names IMO
2. Should we include "airlock" in the azure names as well?
3. Also, are we that limited on those names as well 

Also valid for the workspace level things.

Reply:
fixed